homework_5/Methods/GuessWordGame.java

```diff
@@ -1,5 +1,40 @@
 package Methods;
 
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
 public class GuessWordGame {
-    
+    Scanner scanner = new Scanner(System.in);
+    Random random = new Random();
+    private final String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado" , "broccoli", "carrot", "cherry",
```

Comment:
A class should have a specific order of its members. Constants should be first, static variables second, instance variables third, then constructors, then getters + setters, then public methods, then private methods. Please rearrange your class members accordingly.